src/Sentry/Android/SentrySdk.cs

```diff
@@ -1,52 +1,162 @@
 using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using Android.Content;
 using Android.OS;
 using Android.Runtime;
-using global::Java.Lang;
+using Sentry.Android;
 using Sentry.Internal;
 using Sentry.Protocol;
-using JavaObject = global::Java.Lang.Object;
+using JavaClass = Java.Lang.Class;
+using JavaDouble = Java.Lang.Double;
 
 namespace Sentry
 {
     public static partial class SentrySdk
     {
-        public static IDisposable Init(
-            global::Android.Content.Context context,
-            Action<SentryOptions>? configureOptions)
+        public static IDisposable Init(Context context, Action<SentryOptions>? configureOptions)
         {
             var options = new SentryOptions();
             configureOptions?.Invoke(options);
             return Init(context, options);
         }
 
-        public static IDisposable Init(
-            global::Android.Content.Context context,
-            SentryOptions options)
+        public static IDisposable Init(Context context, SentryOptions options)
         {
             // TODO: Pause/Resume
             options.AutoSessionTracking = true;
             options.IsGlobalModeEnabled = true;
             options.AddEventProcessor(new DelegateEventProcessor(evt =>
             {
-#pragma warning disable 618
-                evt.Contexts.Device.Architecture = Build.CpuAbi;
-#pragma warning restore 618
+                if (Build.SupportedAbis is { } abis)
+                {
+                    evt.Contexts.Device.Architecture = abis[0];
+                }
+                else
+                {
+#pragma warning disable CS0618 // Type or member is obsolete
+                    evt.Contexts.Device.Architecture = Build.CpuAbi;
+#pragma warning restore CS0618 // Type or member is obsolete
+                }
+
                 evt.Contexts.Device.Manufacturer = Build.Manufacturer;
+
                 return evt;
             }));
 
-            Sentry.Android.SentryAndroid.Init(context, new JavaLogger(options),
-                new ConfigureOption(o =>
+            SentryAndroid.Init(context, new JavaLogger(options),
+                new OptionsConfigurationCallback(o =>
                 {
+                    // TODO: Should we set the DistinctId to match the one used by GlobalSessionManager?
+                    //o.DistinctId = ?
```

Comment:
Good point, if we're initializing the dotnet layer first, we could pass down our installation id (will require a new api in java).
In unity we init android first. So during the unity init, we read the installation id from the Java layer and reuse.

Reply:
I'll address this in a future PR.  Thanks.